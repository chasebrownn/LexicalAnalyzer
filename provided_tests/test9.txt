t1a (b).((a)*) #
" baaaaa ba b "