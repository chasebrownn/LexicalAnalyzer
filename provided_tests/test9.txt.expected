t1a , "baaaaa"
t1a , "ba"
t1a , "b"
